eng/build-helpers.psm1

```diff
@@ -57,16 +57,58 @@ function Invoke-Main {
 
     try {
         &$MainBlock
-        Write-Host
-        Write-Host "Build Succeeded" -fore GREEN
+        Write-NewLine
+        Write-Success "Build Succeeded"
         exit 0
     } catch [Exception] {
-        Write-Host
+        Write-NewLine
         Write-Error $_.Exception.Message
-        Write-Host
+        Write-NewLine
         Write-Error "Build Failed"
         exit 1
     }
 }
 
-Export-ModuleMember -Function *
\ No newline at end of file
+#Output message using the Cyan text color
+function Write-Command($message){
+    Write-MessageColorOutput CYAN $message
+}
+
+#Output message using the Green text color
+function Write-Success($message){
+    Write-MessageColorOutput GREEN $message
+}
+
+#Output message using the Yellow text color
+function Write-Info($message){
+    Write-MessageColorOutput YELLOW $message
+}
+
+#Will add a new line
+function Write-NewLine(){
+    Write-MessageColorOutput WHITE "`n"
+}
+
+# Specify one of the following enumerator names
+# Black, DarkBlue, DarkGreen, DarkCyan, DarkRed, DarkMagenta, DarkYellow, Gray, DarkGray, Blue, Green, Cyan, Red, Magenta, Yellow, White"
+function Write-MessageColorOutput($ForegroundColor)
```

Comment:
Three tips for further improvement:

1.  Use [PowerShell style comment blocks](https://learn.microsoft.com/en-us/powershell/scripting/developer/help/syntax-of-comment-based-help?view=powershell-7.4) to provide a description of the function. There are many different elements that can be included in the XML comment descriptions; this article shows the most important one, `.Description`.
2. Use [ValidateSet](https://learn.microsoft.com/en-us/powershell/scripting/developer/cmdlet/validateset-attribute-declaration?view=powershell-7.4) to restrict the allowed strings to those found on line 93.
3. The function declaration is technically correct, but does not follow the preferred PowerShell style. Please see the param block style used in `Invoke-RegenerateFile` above.